### ISSUE TITLE
Charts: highlighting key points such as minimums and maximums

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#453]
Once #429 is complete, one of the first uses of this should be to implement indicators for key points in the chart/graph. Max/min or similar would be an easier first step, but can then be expanded to additional indicators.

**Comment by jeffbl** [comment]
As discussed in tech-arch today, @JRegimbal will not likely have a chance to implement soon, but we should be generating an example or two using this data so that when developer resources come available, we have the design to implement in the handler so that we can move this to production. Putting in Oct3 sprint, but @florian-grond let me know if needs more time.

**Comment by jeffbl** [comment]
@florian-grond let us know if you get a chance to create some examples. Moving to Oct24 since did not happen by Oct3.

**Comment by jeffbl** [comment]
@florian-grond let me know if you're able to start thinking about this. Moving to Nov28 milestone, but please ping me if that isn't feasible.

**Comment by jeffbl** [comment]
Initial experiences were generated and sent on slack. We need to do at least one more design iteration before we'd want to implement these in practice. We're meeting tomorrow morning to start that discussion, but I'm moving this to January for continued effort.

**Comment by jeffbl** [comment]
On hold pending audio/haptic hire
